src/components/admin/hotels/save-hotel/index.js

```diff
@@ -0,0 +1,228 @@
+                                        helperText={formik.touched.email && formik.errors.email}
+                                        variant="outlined"
+                                        size="small"
+                                    />
+                                </Box>
+                                <Box mb={inputMb}>
+                                    <TextField
+                                        fullWidth
+                                        id="phone"
+                                        name="phone"
+                                        placeholder="Phone"
+                                        value={formik.values.phone}
+                                        onChange={formik.handleChange}
+                                        error={formik.touched.phone && Boolean(formik.errors.phone)}
+                                        helperText={formik.touched.phone && formik.errors.phone}
+                                        variant="outlined"
+                                        size="small"
+                                    /></Box>
+                                <Box mb={inputMb}>
+                                    <TextField
+                                        fullWidth
+                                        id="website"
+                                        name="website"
+                                        placeholder="Website"
+                                        value={formik.values.website}
+                                        onChange={formik.handleChange}
+                                        error={formik.touched.website && Boolean(formik.errors.website)}
+                                        helperText={formik.touched.website && formik.errors.website}
+                                        variant="outlined"
+                                        size="small"
+                                    />
+                                </Box>
+                            </Grid>
+                            <Grid item xs={6}>
+                                <Box mb={inputMb}>
+                                    <Typography>
+                                        Attachments:
+                                </Typography>
+                                </Box>
+                                <Box mb={inputMb}>
+                                    <ManageAttachments
+                                        attachments={formik.values.attachments}
+                                        onFileAdd={handleFileAdd}
+                                        onAttachmentRemove={handleAttachmentRemove} />
+                                </Box>
+                            </Grid>
+                        </Grid>
+                    </form>
+                </Container>
+            </DialogContent>
+            <DialogActions>
+                <Grid container direction="column" justify="center" alignItems="center">
```

Comment:
`direction="row" justify="center" alignItems="flex-start"` are redundant for this case.

Reply:
Fixed.